cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -610,82 +610,84 @@ if (BROTLI_VENDORED)
   add_dependencies(brotli_common brotli_ep)
 endif()
 
+if (NOT ARROW_BUILD_PARQUET_DEPS_ONLY)
```

Comment:
Can you replace this with a variable to include LZ4 and ZSTD in the build? I'd rather not see reference to Parquet in this file